pool/database/seed/seed_user.ml

```diff
@@ -243,7 +243,7 @@ let contacts db_label =
   in
   let password =
     Sys.getenv_opt "POOL_USER_DEFAULT_PASSWORD"
-    |> CCOption.value ~default:"user"
+    |> CCOption.value ~default:"Password1!"
```

Comment:
The `admin` password isn't changed, on purpose?

Reply:
Wasn't necessary as we use the Sihl `create_admin` function to seed admins. But I updated this pw as well.